### PR TITLE
Implemented 10 tests on Report Portal.

### PR DESCRIPTION
**Pull request by gbuh**

### CODE REVIEW

**Comment by AkioFaris** [comment]
[19] - c0Mua3f4

**Comment by AkioFaris** [comment]
[19] - Allure report on Netlify: https://5ed5439f7ee67edd1bbfdd61--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
Please change the suite name (currently it's shown as "JDI-Light Bootstrap Tests" in Allure report)

File: src/test/java/io/github/epam/test/data/UserUtils.java [hunk 1]
```diff
@@ -3,5 +3,8 @@
 import io.github.com.entities.User;
 
 public class UserUtils {
-    public static User DEFAULT_USER = new User();
+    public static final User DEFAULT_USER = new User().set(u -> {
+        u.userName = "default";
```
Comment (AkioFaris) [inline src/test/java/io/github/epam/test/data/UserUtils.java]: Remove user's credentials from the code. 
They are passed as command line parameters of the Maven command: "mvn test -Dreport.portal.user=$TEST_USER_LOGIN -Dreport.portal.password=$TEST_USER_PASSWORD" (see travis.yml to understand current Travis CI build)

File: src/test/java/io/github/epam/tests/LoginTest.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import static io.github.com.composites.Header.menuIcon;
+import static io.github.com.composites.Header.userName;
+import static io.github.epam.test.data.UserUtils.DEFAULT_USER;
+
+public class LoginTest extends BaseTest {
+    @Test
+    public void verifyThatUserCanLogonAsDefaultUser() {
+        menuIcon.is().displayed();
+        userName.assertThat().text(DEFAULT_USER.userName.toUpperCase());
```
Comment (AkioFaris) [inline src/test/java/io/github/epam/tests/LoginTest.java:13]: Use System.getProperty("report.portal.user") to get expected username

**Review by AkioFaris — CHANGES_REQUESTED** [review]
1. Resolve comments
2. Please change the suite name (currently it's shown as "JDI-Light Bootstrap Tests" in Allure report)
3. Test should pass in Allure report and on Jenkins (add a link to a passed run in Jenkins)

**Comment by gbuh** [comment]
> 1. Resolve comments
> 2. Please change the suite name (currently it's shown as "JDI-Light Bootstrap Tests" in Allure report)
> 3. Test should pass in Allure report and on Jenkins (add a link to a passed run in Jenkins)

Resolved. Please look into it.

**Comment by AkioFaris** [comment]
[20] - DRHifCsa

**Comment by AkioFaris** [comment]
[20] - Allure report on Netlify: https://5ed60f1764600d45ecd3efa4--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[21] - BxtfmzRK

**Comment by AkioFaris** [comment]
[21] - Allure report on Netlify: https://5ed6154c661e51677220a59d--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[22] - EBQh36aE

**Comment by AkioFaris** [comment]
[22] - Allure report on Netlify: https://5ed615e56afa3c83fb941f0a--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[23] - dmFV4gCt

**Comment by AkioFaris** [comment]
[23] - Allure report on Netlify: https://5ed64b73b63fa95f6886df76--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[24] - ZGKY5D4Y

**Comment by AkioFaris** [comment]
[24] - Allure report on Netlify: https://5ed6a319b1af4b3f393ff33e--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[25] - NEIE9HOf

**Comment by AkioFaris** [comment]
[25] - Allure report on Netlify: https://5ed7773cf9a9bc302f431b3b--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[26] - pKeh5EWS

**Comment by AkioFaris** [comment]
[26] - Allure report on Netlify: https://5ed7f6b905d93879f2ff6649--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[27] - 3R9PevQS

**Comment by AkioFaris** [comment]
[27] - Allure report on Netlify: https://5ed8b2146616f7a6ded83f88--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[28] - mjsienkd

**Comment by AkioFaris** [comment]
[28] - Allure report on Netlify: https://5ed8bf4fc44f696bc1622700--thirsty-yalow-da011e.netlify.app

File: src/main/java/io/github/com/entities/User.java [hunk 3]
```diff
@@ -3,11 +3,9 @@
...
 public class User extends DataClass<User> {
-
-    public String name = System.getProperty("report.portal.user");
+    public String userName = System.getProperty("report.portal.user");
     public String password = System.getProperty("report.portal.password");
 
-//    public String name = "test-user";
-//    public String password = "Fqvq1s0S";
-
+    //    public String userName = "test-user";
```
Comment (AkioFaris) [inline src/main/java/io/github/com/entities/User.java]: Please remove user creds from the code

File: src/main/java/io/github/com/entities/User.java [hunk 4]
```diff
@@ -3,11 +3,9 @@
 import com.epam.jdi.tools.DataClass;
 
 public class User extends DataClass<User> {
-
-    public String name = System.getProperty("report.portal.user");
+    public String userName = System.getProperty("report.portal.user");
```
Comment (AkioFaris) [inline src/main/java/io/github/com/entities/User.java]: Maybe add 2 constructors (with and without parameters)

**Review by AkioFaris — CHANGES_REQUESTED** [review]
See comments

**Comment by AkioFaris** [comment]
[29] - Q4oFVGR2

File: src/main/java/io/github/com/entities/User.java (hunk 4 above)
Comment (gbuh) [reply src/main/java/io/github/com/entities/User.java]: Done

File: src/main/java/io/github/com/entities/User.java (hunk 3 above)
Comment (gbuh) [reply src/main/java/io/github/com/entities/User.java]: Done

**Comment by AkioFaris** [comment]
[29] - Allure report on Netlify: https://5ed92066c29f75c3dc6457bb--thirsty-yalow-da011e.netlify.app

**Review by AkioFaris — APPROVED** [review]

**Review by AkioFaris — APPROVED** [review]

**Comment by AkioFaris** [comment]
[31] - sfKJJxig

**Comment by AkioFaris** [comment]
[31] - Allure report on Netlify: https://5eda12734873ef867bc25ce4--thirsty-yalow-da011e.netlify.app

**Comment by AkioFaris** [comment]
[32] - 2YnylCEC

**Comment by AkioFaris** [comment]
[32] - Allure report on Netlify: https://5eda16027f8e20bd6bd9da09--thirsty-yalow-da011e.netlify.app

**Pull request merged**